DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -60,12 +60,13 @@ public void UpdateLoginCountAndDuration(int progressId)
 		                    FROM Sessions AS S

```

Comment:
The name for this method is no longer accurate. `UpdateProgress` might be a more descriptive name? Remember to also update the test names (so that they include the name of the method they're testing) 🙂